demo/src/main/java/net/minestom/demo/Main.java

```diff
@@ -33,52 +34,53 @@ public class Main {
     public static void main(String[] args) {
         System.setProperty("minestom.experiment.pose-updates", "true");
 
-        MinecraftServer.setCompressionThreshold(0);
+        ServerSettings serverSettings = ServerSettings.builder().compressionThreshold(0).build();
 
-        MinecraftServer minecraftServer = MinecraftServer.init();
+        ServerFacade serverFacade = ServerFacade.of(serverSettings);
```

Comment:
Renaming to ServerFacade is an unnecessary and confusing change imo, unless there is some other reason im missing.